Snakefile

```diff
@@ -201,19 +201,19 @@ rule build_renewable_profiles:
     benchmark: "benchmarks/build_renewable_profiles_{technology}"
     threads: ATLITE_NPROCESSES
     resources: mem_mb=ATLITE_NPROCESSES * 5000
+    wildcard_constraints: technology="solar|onwind|offwind-ac|offwind-dc"
```

Comment:
Can we also do this backward?, i.e. ~hydro? Reason is that it's quite straightforward to add new technologies just through the config.yaml under `renewable`. E.g. you could just provide a new section `offwind-floating` and it would work.

Reply:
Sure, no strong preference there. Updated.